internal/app/app.go

```diff
@@ -22,15 +24,27 @@ type App struct {
 	Handlers   *handlers.Handlers
 	Logger     *logger.ServerLogger
 	FileHelper *utils.FileHelper
+	Context    context.Context
 }
 
 func New() *App {
+	var ns storage.Storage
 	cfg := config.NewConfig()
+	ctx := context.TODO() //определится по требованиям
```

Comment:
```suggestion
	ctx := context.Background() //определится по требованиям
```
В этом месте можно смело стартовать контекст приложения и использовать его потом для мягкой остановки сервиса.

Reply:
Планирую https://habr.com/ru/articles/771626/
Пока мне не все понятно.